xp0-double/xp0d-azuredeploy-msdeploy.json

```diff
@@ -190,9 +190,7 @@
                   "Application Insights Instrumentation Key": "[reference(concat('Microsoft.Resources/deployments/', 'sitecore-infra'), '2015-11-01').outputs.appInsightsInstrumentationKey.value]",
                   "Application Insights Role": "CM",
                   "KeepAlive Url": "[concat('https://', reference(resourceId('Microsoft.Web/sites', variables('cmWebAppNameTidy'))).hostNames[0], '/sitecore/service/keepalive.aspx')]",
-                  "License Xml": "[variables('licenseXml')]",
-                  "IP Security Client IP": "[parameters('security_clientIp')]",
```

Comment:
ah, I see, but no, it's a little bit different: the CM package doesn't have it, because I borrowed the SCWDP from the Single (XP0) setup and that one doesn't have IP-security at all (because if you run that as a single instance, you don't want to secure it on IP). However, I tend to use XP0 single and double in acceptance and test environments only, so I _do_ want IP security.

What I did is I added this IP setting on purpose, and to make sure the SCWDP worked with this ARM templates, I added the IP settings to the SCWDP manually. You could just copy the param in the parameters.xml.